www/styles.css

```diff
@@ -79,7 +79,6 @@ table tr td {
   margin-top: 0;
   margin-bottom: 24px;
   padding: 8px 16px;
-  border-radius: 8px 8px 0px 0px;
```

Comment:
I've removed all use of border-radius because It's kind of a general design rule to make a design fit with your logo and our logo has straight lines and sharp corners.

Reply:
👍